### DISCUSSION TITLE
Error during build: ld: library not found for -lvtkIOCore-9.1

### DISCUSSION THREAD

**Discussion by hsheldon** [idaholab/moose#24305] [Q&A Getting Started] [answered]
I'm getting this error during a build of porous_flow or combined:

ld: library not found for -lvtkIOCore-9.1
clang-12: error: linker command failed with exit code 1 (use -v to see invocation)

Any ideas?

**Comment by hsheldon** [comment, marked as answer]
Ignore that, problem resolved by deleting everything and starting over

**Reply by Bigdogcat** [reply]
Hi, may I know how you solve this problem? I also ran into the same issue: 
ld: library not found for -lvtkParallelMPI-9.2
clang-16: error: linker command failed with exit code 1 (use -v to see invocation)

Thanks very much
